src/components/ConfigSidebar/ConfigDetails.tsx

```diff
@@ -0,0 +1,52 @@
+import { useMemo } from "react";
+import { FaTags } from "react-icons/fa";
+import { useGetConfigByIdQuery } from "../../api/query-hooks";
+import { DescriptionCard } from "../DescriptionCard";
+import { InfoMessage } from "../InfoMessage";
+import { Loading } from "../Loading";
+import Title from "../Title/title";
+
+type Props = {
+  configId: string;
+};
+
+export function ConfigDetails({ configId }: Props) {
+  const {
+    data: configDetails,
+    isLoading,
+    error
+  } = useGetConfigByIdQuery(configId);
+
+  const displayDetails = useMemo(() => {
+    if (configDetails) {
+      return [
+        {
+          label: "Name",
+          value: configDetails.name
+        },
+        ...(configDetails.tags
+          ? Object.entries(configDetails.tags)
+              .filter(([key]) => key !== "Name")
+              .map(([key, value]) => ({
+                label: key,
+                value
+              }))
+          : [])
+      ];
+    }
+  }, [configDetails]);
+
+  return (
+    <div className="flex flex-col space-y-2 max-w-full">
```

Comment:
Can we switch this to a collapsible panel ?

Reply:
Sure